server/app/views/applicant/IneligibleBlockView.java

```diff
@@ -110,6 +132,7 @@ public Content render(
                         messages.at(
                             MessageKey.CONTENT_ELIGIBILITY_CRITERIA.getKeyName(), infoLink)))
                     .withClasses("mb-4"))
+            .condWith(isEligibilityMsgEnabled, div(rawHtml(eligibilityMsg)).withClasses("mb-4"))
```

Comment:
**suggestion:** In addition to checking if the flag is enabled, should we also check that the eligibilityMessage is not an empty string? Wondering if it would cause some weird spacing on the page otherwise

Reply:
Sure! It didn't when i was testing manually, but doesn't hurt to add another check.